.gitignore

```diff
@@ -35,3 +35,6 @@ htmlcov
 
 # pip puts editable packages here
 src
+
+# virtualenv
```

Comment:
I think you can remove this again, right? Or do you want to keep the venv on production in this spot?

Reply:
right :)